lib/main.coffee

```diff
@@ -29,7 +39,7 @@ module.exports =
 
     # Check version of coala
     version = helpers.exec(@executable, ['--version']).then (result) ->
-      if result <= COALA_MIN_VERSION
+      if cmpVersions(result, COALA_MIN_VERSION) < 0
```

Comment:
How about ignoring the `0.` and comparing the rest of the digits which form a float? `9.1` and `10.0`, `10.1`, etc,.? All our versions at the moment follow that norm, in the future when we release 1.x we could update this to keep things simple.